modules/network-observability-pktdrop-overview.adoc

```diff
@@ -21,5 +21,14 @@ When packet drop tracking is enabled, you can see the following metrics represen
 * Top X dropped cause
 * Top X flow dropped rates stacked with total
 
+Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. Dropped flows are shown in the side panel of the *Traffic flows* table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`, and a link to a description of the drop type is also available in the side panel. Examples of host drop reasons are as follows:
```

Comment:
Suggestion to pair up drop types and their prefixes before describing the side panel contents:
```suggestion
Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`. Dropped flows are shown in the side panel of the *Traffic flows* table along with a link to a description of each drop type. Examples of host drop reasons are as follows:
```

Reply:
Good suggestion! I'll take it :)